WorkflowTesting/Sources/RenderExpectations.swift

```diff
@@ -123,15 +124,44 @@ extension ExpectedSideEffect {
 }
 
 public struct ExpectedWorkflow {
-    let workflowType: Any.Type
-    let key: String
-    let rendering: Any
-    let output: Any?
-
-    public init<WorkflowType: Workflow>(type: WorkflowType.Type, key: String = "", rendering: WorkflowType.Rendering, output: WorkflowType.Output? = nil) {
-        self.workflowType = type
-        self.key = key
+    public let rendering: Any
```

Comment:
Exposing the type-erased internals here worries me. Do these need to be public?

Reply:
Actually, I don't think so. Dropping `public` here.